apps/wing-console/console/ui/src/layout/vscode-layout.tsx

```diff
@@ -48,12 +49,19 @@ export const VscodeLayout = ({ cloudAppState, wingVersion }: LayoutProps) => {
     logsRef,
     logs,
     onResourceClick,
+    wingfile,
   } = useLayout({
     cloudAppState,
     defaultLogLevels: ["info", "warn", "error"],
   });
 
-  const { title } = useContext(AppContext);
+  const title = useMemo(() => {
```

Comment:
I think the use-layout should have this logic and return the actual value

Reply:
done